tee-worker/service/src/config.rs

```diff
@@ -61,7 +62,8 @@ pub struct Config {
 	pub running_mode: String,
```

Comment:
Here, these three commands all come from Litentry. They can be put together.

Reply:
Had adjusted.